admin/views/class-view-addons.php

```diff
@@ -113,7 +113,7 @@ public function addon_card( $addon ) { ?>
 
 				<div class="desc column-description">
 
-					<?php echo wpautop( wp_strip_all_tags( $addon->excerpt ) ); ?>
+					<?php echo wp_kses_post( wpautop( wp_strip_all_tags( $addon->excerpt ) ) ); ?>
```

Comment:
I'm fairly certain that `wp_strip_all_tags()` is going to be enough here.  I'm open to hearing about why it's not though.